### PR TITLE
Changelog for 1.10

### PR DESCRIPTION
**Pull request by charliesantos**
Changelog for 1.10

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+
+```
+import { getLogger } from 'loglevel';
+const logger = getLogger(Device.packageName);
+
+// Set log level on subsequent page loads and refreshes
+logger.setLevel('DEBUG');
+```
+
+* sdk.twilio.com is now being used for serving the sound files. (CLIENT-7221)
```
Comment (aymenn) [inline CHANGELOG.md]: I think add https://
Does this resolve to a new IP address?

Comment (charliesantos) [reply CHANGELOG.md]: No, this points to a cloudfront distribution.
I'll add https

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+
+```
+import { getLogger } from 'loglevel';
+const logger = getLogger(Device.packageName);
+
+// Set log level on subsequent page loads and refreshes
+logger.setLevel('DEBUG');
+```
+
+* https://sdk.twilio.com is now being used for serving the sound files. (CLIENT-7221)
```
Comment (aymenn) [inline CHANGELOG.md]: Now I think about it more, we should make a general statement about changing CDN.
What's the plan on updating the instructions for including twilio.js in the changelog?

Comment (charliesantos) [reply CHANGELOG.md]: I was thinking after we release this, we'll update [public changelog](https://www.twilio.com/docs/voice/client/javascript/changelog) with the "Attention" at the top mentioning about the new CDN.

**Comment by charliesantos** [comment]
Was talking with Aymen about 1.10 changelog. We'll copy this over to wagtail and update the debug/warnings section mentioning about loglevel.

**Pull request merged**